installer/x86_64/install.sh

```diff
@@ -92,6 +92,38 @@ VAR_LOG_SIZE=4096
 
 [ -r platforms/$onie_platform ] && . platforms/$onie_platform
 
+# Verify image platform is inside devices list
+# Temporarily set INPUT_TIMEOUT as 10 seconds
+if [ "$install_env" = "onie" ] && [ "$machine" != "generic" ]; then
+    if ! grep -Fxq "$onie_platform" devices/platforms_asic; then
+        INPUT_TIMEOUT=10
+        echo "The image you're trying to install is of a different platform as the running platform"
+        while true; do
+            read -t $INPUT_TIMEOUT -r -p "Do you still wish to install this image? [y/n]: " input
```

Comment:
>$INPUT_TIMEOUT [](http://example.com/codeflow?start=20&length=14)

Actually there is no requirement to timeout. If the script is used by automation, the prompt will be enough to break the normal path (The automation will have its own timeout).

If the script is used by user, it is too rush for them to response in 10 seconds, and without any hint.

Reply:
will remove it